src/layer-manager/lib/generators/basemap/basemap-layers.json

```diff
@@ -0,0 +1,19 @@
+{
+  "north-star": {
+     "id": "north-star",
```

Comment:
Why do you need to repeat the key in the `id` property ?

Reply:
good catch, fixed!